components/processing/form_parser/deployment/main.tf

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+locals {
```

Comment:
This labels are applied for Revenue Tracking.

Reply:
updated